test/utils/verify_dispatch_middleware.dart

```diff
@@ -7,11 +7,13 @@ import 'package:redux/redux.dart';
 ///
 /// [actions] is a static/class member so that it can be accessed in the
 /// anonymous funciton that is created in the class initializer
+///
+/// When writing widget tests, do not rely on state changing as [NextDispatcher] is not being called,
+/// so the real middleware(s) will not be triggered and state will not be updated.
```

Comment:
I think the FakeStore can be used everywhere the `VerifyDispatchMiddleware` was, but I'll leave that for another PR rather than make this one too big.

Reply:
thank you 🙏 😅